<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindowClass</class>
 <widget class="QMainWindow" name="UserWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QListWidget" name="issueListWidget"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="removePushButton">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="addHorizontalLayout">
        <item>
         <widget class="QLineEdit" name="descriptionLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="addPushButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="resolvePushButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Resolve</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>444</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>removePushButton</sender>
   <signal>clicked()</signal>
   <receiver>UserWindowClass</receiver>
   <slot>deleteIssue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addPushButton</sender>
   <signal>clicked()</signal>
   <receiver>UserWindowClass</receiver>
   <slot>addIssue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resolvePushButton</sender>
   <signal>clicked()</signal>
   <receiver>UserWindowClass</receiver>
   <slot>resolveIssue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>deleteIssue()</slot>
  <slot>addIssue()</slot>
  <slot>resolveIssue()</slot>
 </slots>
</ui>
